ui-new/html/limit-reached.html

```diff
@@ -1,24 +1,14 @@
-<section
-  class="viewport limit_reached_state info cash-in-color"
-  data-tr-section="limit-reached"
-  data-screentype="info-action-tp"
->
-  <header>
-    <img src="images/out-of-coins.svg" />
-  </header>
-
-  <main>
-    <h3 class="js-i18n sm-margin-bottom">We're all out of coins!</h3>
-    <p class="js-i18n d2">Please try again later.</p>
-  </main>
-
-  <footer>
-    <button
-      id="limit-reached-ok"
-      display="inline-block"
-      class="filled-progress-button button ok js-i18n"
-    >
-      OK
-    </button>
-  </footer>
+<section class="viewport limit_reached_state cash-out-color" data-tr-section="limit-reached">
```

Comment:
Clearing up something from the chat. The screen that got removed here is still a valid screen called limit_reached_state, the one that got added should be called out_of_cash_state.